hbase-hadoop-compat/src/main/java/org/apache/hadoop/hbase/master/MetricsMasterFilesystemSourceImpl.java

```diff
@@ -72,4 +82,27 @@ public void updateMetaWALSplitTime(long time) {
   public void updateMetaWALSplitSize(long size) {
     metaSplitSizeHisto.add(size);
   }
+
+  @Override
+  public void getMetrics(MetricsCollector metricsCollector, boolean all) {
```

Comment:
Why we need this change? The metrics source does not have a getMetrics method before? Then how does it expose metrics? Strange...